resources/language/resource.language.es_es/strings.po

```diff
@@ -524,7 +528,7 @@ msgstr "Purgar caché en memoria y disco"
 
 msgctxt "#30134"
 msgid "Enable execution timing"
-msgstr "Habilitar temporización de la ejecución"
+msgstr "Habilitar tiempo de ejecución"
```

Comment:
What's the meaning of 'tiempo de ejecución'? Does 'habilitar tiempo' have any meaning in our language (spanish)? 'Tiempo de ejecución' is 'run-time' in english. But timing is to measure time. Hence I preferred 'temporización'. I think this feature is to measure the time of a session.

Reply:
This is used to measure the elapsed time it took to complete some of the functions of the source code

Reply:
@boblo1 https://www.linguee.es/ingles-espanol/traduccion/execution+timing.html

Reply:
> @boblo1
> https://www.linguee.es/ingles-espanol/traduccion/execution+timing.html
> https://www.google.com/search?q=execution+timing&oq=execution+timing&aqs=chrome..69i57j69i60&sourceid=chrome&ie=UTF-8

But 'Habilitar tiempo de ejecución' is not grammatically correct. You cannot 'enable time', but 'enable time measuring'. However, if you use 'Habilitar **medición del** tiempo de ejecución' then the feature name makes more sense. Nevertheless, this alternative text could not fit in the dialog box and text scrolling might happen (undesired effect). That's why I used the noun 'temporización' (shorter than 'medición del tiempo').

Reply:
in Spanish it is not wrong